package.json

```diff
@@ -42,15 +42,18 @@
     "babel-preset-es2015": "^6.24.1",
     "chai": "^3.5.0",
     "eslint": "^3.4.0",
+    "husky": "^0.13.3",
```

Comment:
Thats for automatically running git hooks defined in package.json so its not used anywhere explicitly, but rather it runs itself implicitly